pkg/volume/testing.go

```diff
@@ -227,3 +231,22 @@ func (fd *FakeDeleter) Delete() error {
 func (fd *FakeDeleter) GetPath() string {
 	return fd.path
 }
+
+// FindEmptyDirectoryUsageOnTmpfs finds the expected usage of an empty directory existing on
+// a tmpfs filesystem on this system.
+func FindEmptyDirectoryUsageOnTmpfs() (*resource.Quantity, error) {
+	tmpDir, err := ioutil.TempDir(os.TempDir(), "metrics_du_test")
```

Comment:
So this method is not working around a tmpfs issue, but rather a GCE ext4 issue. Should we rename the method appropriately?